lib/public/views/Runs/Details/editTagsPanel.js

```diff
@@ -17,13 +17,13 @@ import { remoteDataTagPicker } from '../../../components/tag/remoteDataTagPicker
 /**
  * A panel containing:
  * a list of potential tags to add to a RUN
- *a button to update the tag selection
+ * a button to update the tag selection
  * @param {Model} model Pass the model to access the defined functions.
  * @return {vnode} virtual node with representation of the panel
  */
 const editTagsPanel = (model) => h(
     '#tags-selection.flex-column.w-30',
-    remoteDataTagPicker(model.tags.getTags(), model.runs.detailsModel.editionTagPickerModel, { outlineSelection: true }),
+    remoteDataTagPicker(model.tags.getActiveTags(), model.runs.detailsModel.editionTagPickerModel, { outlineSelection: true }),
```

Comment:
It would be great if the page for editing the tags of a run displays also in `()` that the tag is archived so that the user knows if they deselect it, they will not be able to add it back to the run

Reply:
Done